README.md

```diff
@@ -1,2 +1,99 @@
 # testing
-Shared testing framework for Source projects
+Shared testing framework for Source projects.
+
+This repository provides a shared language for writing tests across Source Network projects.  Tests are composed entirely of `action.Action`s, `multiplier.Multiplier`s may be configured to mutate the declared actions to scale with various project-specific complexity multipliers.
```

Comment:
todo: The flow of the second sentence is a little weird. It feels like the comma should be a period. Please fix it 🙂.